common/src/Microsoft.Azure.IIoT.Hub.Module.Framework/src/Hosting/MethodRouter.cs

```diff
@@ -99,7 +99,7 @@ public sealed class MethodRouter : IMethodRouter, IMethodHandler {
         /// <param name="target"></param>
         private void AddToCallTable(object target) {
             var versions = target.GetType().GetCustomAttributes<VersionAttribute>(true)
-                .Select(v => "_v" + v.Value)
```

Comment:
Is direct method name comparison case-sensitive or not ? I am asking because `v` here is lowercase while the `V` in the new values in all controllers is uppercase.

Reply:
I see that in our E2E tests we have used uppercase `V` and it worked, so the comparison should be case-insesitive.

Reply:
it's insensitive